packages/form/src/components/UnitInputField/index.tsx

```diff
@@ -0,0 +1,111 @@
+import { UnitInput } from '@ultraviolet/ui'
+import type { ComponentProps } from 'react'
+import { useController } from 'react-hook-form'
+import type { FieldPath, FieldValues } from 'react-hook-form'
+import { useErrors } from '../../providers'
+import type { BaseFieldProps } from '../../types'
+
+type UnitInputFieldProps<
+  TFieldValues extends FieldValues,
+  TName extends FieldPath<TFieldValues>,
+> = BaseFieldProps<TFieldValues, TName> &
+  Pick<
+    ComponentProps<typeof UnitInput>,
+    | 'id'
+    | 'name'
+    | 'className'
+    | 'data-testid'
+    | 'disabled'
+    | 'value'
+    | 'max'
+    | 'min'
+    | 'options'
+    | 'placeholder'
+    | 'placeholderUnit'
+    | 'size'
+    | 'unitValue'
+    | 'required'
+    | 'width'
+    | 'helper'
+    | 'selectInputWidth'
+  > & {
+    onChange?: ComponentProps<typeof UnitInput>['onChange']
+    onChangeUnitValue?: ComponentProps<typeof UnitInput>['onChangeUnitValue']
+    label: string
```

Comment:
You need to add the validate prop as we have in other Fields (like TextInputField)